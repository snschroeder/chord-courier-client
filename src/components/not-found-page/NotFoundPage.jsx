import React from 'react';

export default function NotFoundPage() {
  return (
    <h1>
      Oh dear, something went wrong.
    </h1>
  );
}
