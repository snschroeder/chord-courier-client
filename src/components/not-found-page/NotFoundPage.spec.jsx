/* eslint-disable no-undef */
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { BrowserRouter } from 'react-router-dom';
import NotFoundPage from './NotFoundPage';

describe('NotFoundPage component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <BrowserRouter>
        <NotFoundPage />
      </BrowserRouter>,
      div,
    );
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders the header correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
