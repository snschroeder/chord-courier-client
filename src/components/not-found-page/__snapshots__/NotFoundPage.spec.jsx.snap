// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component renders the header correctly 1`] = `
<h1>
  Oh dear, something went wrong.
</h1>
`;
